rtscomponentkit/src/main/java/io/dolby/rtscomponentkit/domain/StreamingData.kt

```diff
@@ -18,19 +24,51 @@ data class ConnectOptions(
     companion object {
         fun from(
             useDevEnv: Boolean,
```

Comment:
I think this useDevEnv value can be removed

Reply:
I kept it so that when someone updates the app, it won't cause an issue